### DISCUSSION TITLE
GitHub is great!!

### DISCUSSION THREAD

**Discussion by richi-rm** [community/community#57762] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I love GitHub.

**Comment by DanielLuzhub** [comment]
Estou começando agora no Github, espero aprender muito estando aqui com tantas cabeças pensantes

**Comment by zxaylex** [comment]
Yes! I've been scrolling through github more than facebook all day

**Comment by RoshdiRaed** [comment]
Github is a great admin, and very helpful 🤍🤍

**Reply by Roshdi2** [reply]
Nics

**Reply by Roshdi2** [reply]
Good
